src/Arcus.Observability.Tests.Integration/Serilog/ApplicationInsightsSinkTests.cs

```diff
@@ -121,6 +121,30 @@ public void LogRequest_SinksToApplicationInsights_ResultsInRequestTelemetry()
             }
         }
 
+        [Fact]
+        public void LogDependency_SinksToApplicationInsights_ResultsInDependencyTelemetry()
+        {
+            // Arrange
+            var configuration = new LoggerConfiguration().WriteTo.AzureApplicationInsights(_instrumentationKey);
+            using (var loggerFactory = LoggerFactory.Create(builder => builder.AddSerilog(configuration.CreateLogger(), dispose: true)))
+            {
+                ILogger logger = loggerFactory.CreateLogger<ApplicationInsightsSinkTests>();
+
+                string dependencyType = _bogusGenerator.Name.FullName();
+                object dependencyData = _bogusGenerator.Finance.Account();
+                bool isSuccessful = _bogusGenerator.PickRandom(true, false);
+                DateTimeOffset startTime = _bogusGenerator.Date.PastOffset();
+                TimeSpan duration = _bogusGenerator.Date.Timespan();
+                Dictionary<string, object> telemetryContext = CreateTestTelemetryContext();
+
+                // Act
+                logger.LogDependency(dependencyType, dependencyData, isSuccessful, startTime, duration, telemetryContext);
```

Comment:
Did you verify it was tracked in Application Insights?

Reply:
Yes, it was not tracked.

Reply:
Just like the requests are not tracked.

Reply:
Then we have to fix the tests before we merge it 🤷‍♂️